src/dialog-utilities.js

```diff
@@ -0,0 +1,39 @@
+import {Origin} from 'aurelia-metadata';
+
+/**
+* Returns a view model based on a given instruction
+*
+* @param {Object} instruction the instruction from which the view model is determined
+* @return {Promise} the view model
+*/
+export function getViewModel(instruction: any, compositionEngine: CompositionEngine) {
```

Comment:
I may be mistaken, but this should still be private to the module. I'm not exporting it in `index.js`. Agreed though. As I mentioned above though, this method was already not fully tested in that the `compositionEngine` branch was never triggered